root/plex-common.sh

```diff
@@ -1,5 +1,25 @@
 #!/bin/bash
 
+CONT_CONF_FILE="/version.txt"
+
+function addVarToConf {
+  local variable="$1"
+  local value="$2"
+  if [ ! -z "${variable}" ]; then
+    echo ${variable}=${value} >> ${CONT_CONF_FILE}
+  fi
+}
+
+function readVarFromConf {
+  local variable="$1"
+  declare -n value=$2
+  if [ ! -z "${variable}" ]; then
+    value="$(grep -w ${variable} ${CONT_CONF_FILE} | cut -d'=' -f2 | tail -n 1)"
+  else
+    value=NULL
+  fi
+}
+
```

Comment:
@gbooker I think there are better ways to write different variables from a "conf" file, but I wanted to keep things simple given that these variables are written once in this file (while building the image), then they are read only.

Reply:
Since the values are written once and in only one place, the function to add them isn't really needed (doesn't hurt though).  I was expecting just a
```
echo "version=${TAG}" > /version.txt
echo "plex_build=${PLEX_BUILD}" >> /version.txt
echo "plex_distro=${PLEX_DISTRO}" >> /version.txt
```
but the above should work too.

Reply:
Yes exactly I was thinking to write the same lines at the beginning like you are suggesting. At least if at some point we need to have such as a function, we just have to modify the behaviour of this one